<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cassandra View</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icon</normaloff>:/icons/icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Host:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="f_connectionBtn">
              <property name="text">
               <string>localhost:9160</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="contextLabel">
            <property name="text">
             <string>Snap! Context:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="f_contextEdit">
            <property name="text">
             <string>snap_websites</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="tablesLabel">
            <property name="text">
             <string>Tables:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="f_tables"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelFilter">
        <property name="text">
         <string>Filter:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="f_filterEdit">
        <property name="toolTip">
         <string>Enter a regular expression to filter the rows.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_applyFilter">
        <property name="toolTip">
         <string>Filter rows with regular expression.</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/find_icon</normaloff>:/icons/find_icon</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_clearFilter">
        <property name="toolTip">
         <string>Clear regex filter.</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/clear_icon</normaloff>:/icons/clear_icon</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_refreshView">
        <property name="toolTip">
         <string>Force a refresh of the entire view.</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/reload_icon</normaloff>:/icons/reload_icon</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="f_mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="f_rowsGroup">
       <property name="title">
        <string>Rows</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListView" name="f_rowsView">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="f_cellsGroup">
       <property name="title">
        <string>Columns</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListView" name="f_cellsView">
          <property name="selectionMode">
           <enum>QAbstractItemView::ContiguousSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="f_valueGroup">
       <property name="title">
        <string>Value</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="f_valueEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Settings"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Helpo">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_Column">
    <property name="title">
     <string>&amp;Column</string>
    </property>
    <addaction name="action_InsertColumn"/>
    <addaction name="action_DeleteColumns"/>
   </widget>
   <widget class="QMenu" name="menu_Row">
    <property name="title">
     <string>Ro&amp;w</string>
    </property>
    <addaction name="action_InsertRow"/>
    <addaction name="action_DeleteRows"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Row"/>
   <addaction name="menu_Column"/>
   <addaction name="menu_Helpo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Settings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="action_InsertColumn">
   <property name="text">
    <string>&amp;Insert Column</string>
   </property>
   <property name="toolTip">
    <string>Insert new column into the current row.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_DeleteColumns">
   <property name="text">
    <string>&amp;Delete Columns</string>
   </property>
   <property name="toolTip">
    <string>Delete selected columns in the current row.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_InsertRow">
   <property name="text">
    <string>&amp;Insert Row</string>
   </property>
   <property name="toolTip">
    <string>Insert new row into current table.</string>
   </property>
  </action>
  <action name="action_DeleteRows">
   <property name="text">
    <string>&amp;Delete Rows</string>
   </property>
   <property name="toolTip">
    <string>Delete selected rows in current table.</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
